src/store/connectionStore.ts

```diff
@@ -212,5 +274,24 @@ export const useConnectionStore = defineStore('connectionStore', {
 
       return transformToCurl({ method, headers, url, ssl: sslCertVerification, qdsl });
     },
+    async testDynamoDBConnection(connection: DynamoDBConnection) {
+      const client = new DynamoDBClient({
+        region: connection.region,
+        credentials: {
+          accessKeyId: connection.accessKeyId,
+          secretAccessKey: connection.secretAccessKey,
+        },
+      });
+      
+      try {
+        await client.send(new ListTablesCommand({}));
+        return true;
+      } catch (error) {
+        if (error instanceof Error) {
+          throw new Error(error.message);
+        }
+        throw new Error('Unknown error occurred while testing connection');
+      }
```

Comment:
@aws-sdk/client-dynamodb should not able to run here, since all ts/vue content running in browser rather than nodejs, so to call the dynamodb,

vue -> rust dynamo lib -> dynamodb

Reply:
Oh, yes! You are right! I will update soon

Reply:
Where is our backend service? Vue will call rust api via axios http. I'm not sure where rust backend is right now. Can I subsequently modify the logic to connect to this?